hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/QuantileSketch.java

```diff
@@ -58,7 +53,12 @@ public QuantileSketch(final RecordOrder sortOrder, final int resolution) {
      */
     public QuantileList getQuantile(final Table data) {
         /* Sample a set of rows from the table, then sort the sampled rows. */
-        final IMembershipSet sampleSet = data.members.sample(this.resolution * this.perBin);
+        /*
+      a knob to control the sample size taken fromm a table to create a QuantileList
```

Comment:
Comment spacing.
Please make perBin a static final field of the class (assuming it does not change), and the comment a JavaDoc comment.